### PR TITLE
Implement UUPS Proxy in BorrowerOperations.sol and refactor helpers

### PR DESCRIPTION
**Pull request by arthurka-o**

### CODE REVIEW

File: packages/contracts/contracts/BorrowerOperations.sol [hunk 1]
```diff
@@ -96,6 +96,16 @@ contract BorrowerOperations is KumoBase, CheckContract, IBorrowerOperations {
     
     // --- Dependency setters ---
 
+    function initialize() external initializer {
+        __Ownable_init();
+        __UUPSUpgradeable_init();
+    }
+
+    /// @custom:oz-upgrades-unsafe-allow constructor
```
Comment (udosson) [inline packages/contracts/contracts/BorrowerOperations.sol:104]: Can we remove these comments?

File: packages/contracts/contracts/SortedTroves.sol [hunk 2]
```diff
@@ -86,6 +86,11 @@ contract SortedTroves is Initializable, UUPSUpgradeable, OwnableUpgradeable, Che
         __UUPSUpgradeable_init();
     }
 
+    /// @custom:oz-upgrades-unsafe-allow constructor
```
Comment (udosson) [inline packages/contracts/contracts/SortedTroves.sol:89]: Can we remove these comments?

File: packages/contracts/contracts/StabilityPool.sol [hunk 3]
```diff
@@ -278,10 +278,10 @@ contract StabilityPool is KumoBaseUpgradeable, CheckContract, IStabilityPool {
         P = DECIMAL_PRECISION;
     }
 
-    /// @custom:oz-upgrades-unsafe-allow constructor
-    constructor() {
-        _disableInitializers();
-    }
+    //// @custom:oz-upgrades-unsafe-allow constructor
```
Comment (udosson) [inline packages/contracts/contracts/StabilityPool.sol:281]: Same here

**Review by udosson — APPROVED** [review]
Very good work, @arthurka-o !
Please check my comments and remove the commented section if not needed. 
Then, please go ahead and merge this PR

**Comment by arthurka-o** [comment]
I removed initializing constructors, but we must implement some initialization for implementation contracts from the JS side. 
I'll create an issue for this.

**Review by udosson — APPROVED** [review]
Ready to go

**Pull request merged**